### ISSUE TITLE
Accessing conda environment packages in Jupyter notebooks 

### ISSUE THREAD

**Issue by laurahspencer** [RobertsLab/resources#935]
I've installed a couple programs (Salmon, mgkit) in a conda environment called "env_bioinf". I am able to access and run packages via `conda activate env_bioinf` in the terminal, but can't figure out how to do so in Jupyter notebooks. Here's what I've tried: 

I followed [this blog post](https://medium.com/@nrk25693/how-to-add-your-conda-environment-to-your-jupyter-notebook-in-just-4-steps-abeab8b8d084) I set up a conda environment called "env_bioinf" and added it as a kernel to my jupyter notebook. 

```
    conda create --name env_bioinf
    conda activate env_bioinf
    conda install -c conda-forge mgkit
    conda install -c conda-forge salmon #I need that for another part of my pipeline, so installed that in the env_bioinf env too 
    conda install -c anaconda ipykernel
    python -m ipykernel install --user --name=env_bioinf
```
That all worked great as I can successfully switch to my new "env_bioinf" kernel: 
![image](https://user-images.githubusercontent.com/17264765/82086294-28689680-96a3-11ea-87f9-a5115d12d767.png)

_However_ I still cannot access the packages that I need (`blast2gff`, from mgkit) within Jupyter even when I run my notebook with my new env_bioinf kernel. I can, however, view and use `blast2gff` by using the paths. 

![image](https://user-images.githubusercontent.com/17264765/82087495-1687f300-96a5-11ea-923a-aa8749271e8e.png)
![image](https://user-images.githubusercontent.com/17264765/82087546-2b648680-96a5-11ea-8bd2-aaab96c03832.png)

**Any ideas how to make Jupyter notebook access the conda environment's packages?** 

### Other attempt 
I also tried to activate my env_bioinf conda environment _within_ Jupyter notebook via `conda activate env_bioinf, and got this error: 

```
CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.
```

I followed the directions to run `conda init bash`, closed all notebooks/terminals, re-tried to activate, and got the same error.  I see that Natalie had a [similar issue with conda environments](https://github.com/RobertsLab/resources/issues/845) on Mox (although it wasn't related to Jupyter notebooks).

**Comment by laurahspencer** [comment]
I can, indeed, access `blast2gff` via the full path. But, my impression was that by installing packages in a conda environment, then activating that environment, I do not need to add each program to my $PATH.  This indeed is how it works when I'm within the Terminal, but not within Jupyter notebooks.

**Comment by laurahspencer** [comment]
Oh, I get it now. Yes, my "env_bioinf" environment is in my $PATH 

```
echo $PATH

/usr/local/opt/icu4c/sbin:/usr/local/opt/icu4c/bin:/Users/laura/anaconda3/envs/env_bioinf/bin:/Users/laura/anaconda3/condabin:/anaconda3/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin
```

**Comment by laurahspencer** [comment]
But interestingly, when I pull up my $PATH in Jupyter notebook the env_bionf directory is not there: 

![image](https://user-images.githubusercontent.com/17264765/82090165-e68f1e80-96a9-11ea-9b43-f7d2f86971a4.png)

**Comment by kubu4** [comment]
I played with this for a bit this morning. Here's what I did and things work without issue. Maybe give these steps a try (with different environment name - or delete old `env_bioinf` environment first)?

1. `conda create --name env_bioinf`

2. `conda activate env_bioinf`

3. `conda install -c bioconda mgkit` (NOTE: conda didn't find this in the conda-forge channel you listed in your original post)

4. Launch Jupyter notebook.

5. Select a Python kernel.

That's it. Everything should work as expected. No need for the `ipykernal` install steps.

FYI, this was done with `conda 4.8.3`.

---

![image](https://user-images.githubusercontent.com/4514104/87684860-9713b000-c737-11ea-951b-979802510ea4.png)
